### PR TITLE
Add gopher url

### PR DESCRIPTION
**Pull request by PhilmacFLy**
As Gopher is a fast-growing emerging technology, I find it important that a space can also show his representation in the gopher space on the space API

### CODE REVIEW

**Comment by PhilmacFLy** [comment]
I guess with the url array would really be the better solution

**Comment by rnestler** [comment]
> Maybe the url field should be an array and provide which protocol you should use?

Changing the URL from a string to an array would be a breaking change. (Also in practice the url field means website over http in most of the cases and is not used for any other protocol)

**Comment by dbrgn** [comment]
I think gopher fits in well alongside twitter and google+.

File: 14-draft.json [hunk 1]
```diff
@@ -264,6 +264,10 @@
         "issue_mail": {
           "description": "A separate email address for issue reports (see the <em>issue_report_channels</em> field). This value can be Base64-encoded.",
           "type": "string"
+        },
+        "gopher": {
+          "description": "A url to find information about the Space in the Gopherspace.",
```
Comment (rnestler) [inline 14-draft.json]: Spelling: "A URL"

Also maybe add an explicit example like: gopher://whatever

**Comment by dbrgn** [comment]
Ping @PhilmacFLy, do you plan to implement a fix for the suggestion by @rnestler?

**Comment by PhilmacFLy** [comment]
Sorry must have missed the suggestion aswell as the first ping. Yes I will do a fix for it.

**Pull request closed without merging**